src/NewSightingSubmission.jsx

```diff
@@ -0,0 +1,81 @@
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+
+const NewSightingSubmission = () => {
+  const [newSighting, setNewSighting] = useState({
+    date: "",
+    location: "",
+    notes: "",
+  });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    //on Submit, make a post request to the backend.
+    const sendData = async () => {
+      try {
+        //make a http POST request to the backend.
+        let response = await axios.post(
+          `${import.meta.env.VITE_SOME_BACKEND_URL}/sightings`,
+          newSighting
+        );
+
```

Comment:
When you send your new post you can retrieve all of the sightings from your backend and update your applications state from there
Then you dont need a refresh button